src/applications/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -110,7 +109,7 @@ export class VeteranBenefitSummaryLetter extends React.Component {
       }
     });
 
-    const vaBenefitInformation = (
+    const vaBenefitInformation = vaBenefitInfoRows.length ? (
```

Comment:
do you think it is worth using `vaBenefitInfoRows?.length` to catch an edge case of it being undefined or null?

Reply:
otherwise LGTM!

Reply:
oh sorry, I missed your question. It's defined on line 68 as an empty array. So there shouldn't be an edge case.